packages/cosmwasm-stargate/src/signingcosmwasmclient.ts

```diff
@@ -95,6 +101,10 @@ function createBroadcastTxErrorMessage(result: BroadcastTxFailure): string {
 function createDefaultRegistry(): Registry {
   return new Registry([
     ["/cosmos.bank.v1beta1.MsgMultiSend", MsgMultiSend],
+    ["/cosmos.distribution.v1beta1.MsgFundCommunityPool", MsgFundCommunityPool],
+    ["/cosmos.distribution.v1beta1.MsgSetWithdrawAddress", MsgSetWithdrawAddress],
+    ["/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward", MsgWithdrawDelegatorReward],
+    ["/cosmos.distribution.v1beta1.MsgWithdrawValidatorCommission", MsgWithdrawValidatorCommission],
```

Comment:
It would be nice if we could get this list from `@cosmjs/stargate` and add the wasm types to it. The types are coming from `@cosmjs/stargate` already via

 ```
import {
  AminoTypes,
  BroadcastTxFailure,
  BroadcastTxResponse,
  codec,
  isBroadcastTxFailure,
  parseRawLog,
} from "@cosmjs/stargate";
```